solutions/silver/cf-818F.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: cf-818F
+source: CF
+title: Level Generation
+author: Justin Ji
+---
+
+<Spoiler title="Hint 1">
+
+The upper bound for the number of bridges we can have is $n - 1$ because
+each bridge is present in any spanning tree of the graph. A
+spanning tree is a subgraph that connects every node without cycles.
```

Comment:
maybe just link to the wikipedia or brilliant defn

Reply:
good point